homeworks/nagynorbert/2_vars_datatypes/user_info.py

```diff
@@ -0,0 +1,73 @@
+# 2. homework - Nagy Norbert
+from pprint import pprint
+
+user_info = {
+    "name": "Mike",
+    "age": 25,
+    "favourite_meals": [
+        "pizza",
+        "carbonara",
+        "sushi"
+    ],
+    "phone_contacts": {
+        "Mary": "+36701234567",
+        "Tim": "+36207654321",
+        "Tim2": "+36304567321",
+        "Jim": "+364005000"
+    }
+}
+
+# 1.
+input_skills = input("Please add 4 skills with comma separated format:")
+user_info.update({"skills":input_skills.split(",")})
+pprint(user_info)
+
+# 2.
+user_info["favourite_meals"].sort()
+
+# 3.
+pprint(user_info["favourite_meals"][-2])
+
+# 4.
+user_info["favourite_meals"].append("spaghetti")
+pprint(user_info)
+
+# 5.
+user_info["favourite_meals"].extend(user_info["favourite_meals"][2:4])
+pprint(user_info)
+
+# 6.
+fav_meals_l = user_info["favourite_meals"]
+fav_meals_no_dupl_l = list(set(fav_meals_l))
+fav_meals_no_dupl_l.sort()
+user_info["favourite_meals"] = fav_meals_no_dupl_l
+pprint(user_info)
+
+# 7.
+user_info["favourite_meals"][0] = "rantott_hus"
```

Comment:
Itt a feladat az lenne, hogy úgy cseréld fel az első és utolsó elemet, hogy nem az értékükre hivatkozol, hanem az index-ükre. Ha itt nem rántott hús és somlói van a listában, a mostani kóddal akkor is ezeket az értékeket adnád meg, és az már nem felcserélés, hanem szimplán értékadás.